ui/lib/kubernetes/addon/components/page/configure.hbs

```diff
@@ -49,8 +49,8 @@
     <Icon @name="check-circle-fill" class="has-text-green" />
     <span>Configuration values were inferred successfully.</span>
   {{else if (eq this.inferredState "error")}}
```

Comment:
issue with duplicate helper classes. has-text-danger is now the helper class that color: red !important.